tests/test_common.py

```diff
@@ -570,9 +523,7 @@ def test_string_arrays():
 def test_clear_arrays():
     # First try on an empty mesh
     grid = pyvista.UniformGrid((10, 10, 10))
-    grid.clear_arrays()
     # Now try something more complicated
-    grid = GRID.copy()
```

Comment:
Just noticed this.  Any reason why we change the grid that we're using here?  We start off with a uniform grid and then change it to our hex example above.  Maybe split up the test so we check clearing the array on both types?  Would `hypothesis` be a good case for this?

Reply:
Functionally, the difference is that the `UniformGrid` is already empty, whereas `grid` already contains scalars.  Maybe a separate test that clears it twice?

Reply:
We can do this on a separate PR and merge this now.